### PR TITLE
minikube updates

### PR DESCRIPTION
**Pull request by cmluciano**

### CODE REVIEW

File: _docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -32,7 +32,15 @@ support).
 
 * Depending on your Kubernetes provider:
 
-### [Minikube](https://github.com/kubernetes/minikube/releases)
+  * To install Istio locally, install the latest version of
+[Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) (version 0.25.0 or later).
+  ```
```
Comment (geeknoid) [inline _docs/setup/kubernetes/quick-start.md]: Please add "bash" after the triple ticks.

**Comment by geeknoid** [comment]
Should we go ahead and merge this?

File: _docs/setup/kubernetes/quick-start.md (hunk 1 above)
Comment (cmluciano) [reply _docs/setup/kubernetes/quick-start.md]: I will update this, sec

**Comment by cmluciano** [comment]
This has already been fixed, closing

**Pull request closed without merging**